Sources/DefaultPlugins/DebooggerButtonPlugin.swift

```diff
@@ -0,0 +1,44 @@
+//
+// Copyright (c) 2019 Rosberry. All rights reserved.
+//
+
+import Foundation
+
+final class DebooggerButtonPlugin: SwitchPlugin {
+
+    private enum Constants {
+        static let debooggerSuffix: String = "deboogger_"
+    }
```

Comment:
Actually, it'a not a suffix, it's a prefix 🤓

Reply:
😱